src/components/ProjectSection/ProjectData.ts

```diff
@@ -143,13 +143,13 @@ export const CurrentProjects: Project[] = [
   },
   {
     id: 3,
-    name: "LifeLine",
-    github: "https://github.com/techstartucalgary/LifeLine",
-    pmName: "Tim Macphail",
+    name: "Lifeline",
+    github: "https://lifeline.techstartucalgary.com/",
```

Comment:
Please keep this as your project's github repo link, and incldue the deployed url in your project README. Thanks!